src/InvalidUrlView.js

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2021 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import {TemplateView} from "./utils/TemplateView.js";
+import {LinkKind} from "./Link.js";
+
+export class InvalidUrlView extends TemplateView {
+    render(t, vm) {
+        return t.div({ className: "DisclaimerView card" }, [
+            t.h1("Invalid URL"),
+            t.p([
+                'The link you have entered is not valid. If you like, you can ',
+                t.a({ href: "#/" }, 'return to the home page.')
+            ]),
+            vm.validFixes.length ? this._renderValidFixes(t, vm.validFixes) : [],
+        ]);
+    }
+
+    _describeLinkKind(kind) {
+        switch (kind) {
+            case LinkKind.Room: return "The room ";
```

Comment:
Perhaps look at `link.identifierKind`to differentiate between room and room alias here? Wdyt?

Reply:
Only thing is, I'm not sure how to make the distinction in "plain terms". Do we want to ask users about _room aliases_? If I click on an invite to `#thisweekinmatrix` and I'm told it's an alias, that seems a little weird.

Reply:
I think it's more confusing to give two options labeled "The room". Perhaps label a room id "The room" and a room alias "The room alias"?

Reply:
Implemented this in 3802829 :)